corehq/apps/app_manager/suite_xml/features/mobile_ucr.py

```diff
@@ -95,9 +95,9 @@ def get_custom_entries(self):
 
 def _get_config_entry(config, domain, new_mobile_ucr_restore=False):
     if new_mobile_ucr_restore:
-        datum_string = "instance('commcare-reports:{}')/rows"
+        nodeset = "instance('commcare-reports:{}')/rows".format(config.instance_id)
     else:
-        datum_string = "instance('reports')/reports/report[@id='{}']"
+        nodeset = "instance('reports')/reports/report[@id='{}']".format(config.uuid)
```

Comment:
oh nice, I'd say this is even a strict improvement over the earlier version - I was expecting a cumbersome workaround.